packages/@angular/cli/utilities/check-package-manager.ts

```diff
@@ -19,16 +19,16 @@ export function checkYarnOrCNPM() {
       .then((data: Array<boolean>) => {
         const [isYarnInstalled, isCNPMInstalled] = data;
         if (isYarnInstalled && isCNPMInstalled) {
-          console.log(chalk.yellow('You can `ng set --global packageManager=yarn` '
-            + 'or `ng set --global packageManager=cnpm`.'));
+          console.log(chalk.yellow('You can `ng config -g cli.packageManager \'yarn\'` '
```

Comment:
The quotes (single or otherwise) are no longer needed.

For example, `ng config -g cli.packageManager yarn` will work with rc.7+.